scopesim/effects/psf_utils.py

```diff
@@ -109,9 +109,9 @@ def make_strehl_map_from_table(tbl, pixel_scale=1*u.arcsec):
     return map_hdu
 
 
-def rescale_kernel(image, scale_factor, spline_order=None):
+def rescale_kernel(image, scale_factor, spline_order=None, cmds=None):
```

Comment:
I appreciate why this is here, but I would prefer to keep `cmds `out of the utils functions. Maybe it's just me being pedantic for no reason, but the util function should be able to be called by anything, even if a `cmds `object doesn't exist.

I would suggest deleting the call to `currsys `instead and forcing the offending `Effect `object to resolve the `!`-string before calling `rescale_kernel`

```suggestion
def rescale_kernel(image, scale_factor, spline_order):
```